change-beta/@azure-communication-react-358e8049-76ac-4652-bd8b-a80c781827e7.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
```

Comment:
I would suggest we don't have changelog entries for PR, and instead just have one big changelog entry where we have shifted to storybook v8

Reply:
for sure let me remove them thanks!